### ISSUE TITLE
new bots impossible #STOP BOTS WON AND WE LOSE 

### ISSUE THREAD

**Issue by ghost** [davidmann4/ogar-feeder-bot#555]
agar.io protocol 100% changed

**Comment by hassanila** [comment]
Yes, very possible to fix them but it will take a long time.. I just checked the "deobfuscated code" and I can see that it can be fixed.

**Comment by Nick8fr32** [comment]
so there is still hope out there @hassanila97

**Comment by hassanila** [comment]
Yes, youll see bots everywhere very soon  and I have extracted some very useful new code.

**Comment by Nick8fr32** [comment]
are u going to share or do u keep it all private??

**Comment by hassanila** [comment]
I will share a pic for helping but my code will be private.

**Comment by Nick8fr32** [comment]
okay

**Comment by hassanila97** [comment]
But my laptop 8gb ram 3.02ghz quad-core Windows 10 cant handle all those 40.000+ lines of code.. Its lagging as fuck

**Comment by Nick8fr32** [comment]
i will try to look into it

**Comment by yario-o** [comment]
@hassanila97 I hope you will success

**Comment by hassanila** [comment]
A hint for all of you is

 `var s = {`   and   `s.sockets`  

ONLY ON DEOBFUSCATED CODE OFC

**Comment by yario-o** [comment]
@hassanila97 Which JS file you deobfuscated?
And what do you use for deobfucast the code?

**Comment by hassanila** [comment]
`agario.core.js`   but the info above is useful to only those that can code...

---

**HINT**

![2016-05-07 19_22_43-json2 js c__users_hassan_desktop_json2 js atom](https://cloud.githubusercontent.com/assets/15068696/15093458/28e812ca-1489-11e6-9f8f-98e655e31cbd.png)

**Comment by yario-o** [comment]
@hassanila97 thx
